files/zh-cn/web/api/navigator/clipboard/index.md

```diff
@@ -29,9 +27,9 @@ navigator.clipboard
   );
 ```
 
-这个代码片段将 HTML 中拥有类名 `"cliptext"` 的第一个元素的内容替换为剪切板中的内容。这段代码可用于在浏览器拓展中定时自动更新或者由事件触发，实时显示当前剪切板上的内容。
+该代码片段将层叠样式表为 `“cliptext”` 的元素的内容替换为剪贴板中的文本内容。这段代码可能用于浏览器扩展程序，该扩展程序显示当前剪贴板的内容，并会定期自动更新或在特定事件触发时更新。
 
-如果剪切板为空，或者不包含文本，则 `"cliptext"` 元素的内容将被清空。这是因为在剪切板为空或者不包含文本时，{{domxref("Clipboard.readText", "readText()")}} 会返回一个空字符串。
+如果剪贴板为空或不包含文本，则清除 `“cliptext”` 元素的内容。这是因为如果剪贴板为空或不包含文本，否则 {{domxref("Clipboard.readText", "readText()")}} 会返回空字符串。
```

Comment:
Please do not replace quotes in inline code with full-width quotes as it represents a JavaScript/CSS/HTML string.

```suggestion
如果剪贴板为空或不包含文本，则清除 `"cliptext"` 元素的内容。这是因为如果剪贴板为空或不包含文本，{{domxref("Clipboard.readText", "readText()")}} 会返回空字符串。
```

Reply:
OK